forge16/src/main/java/com/envyful/poke/removal/forge/config/PokeRemovalConfig.java

```diff
@@ -23,7 +22,7 @@ public class PokeRemovalConfig extends AbstractYamlConfig {
     private Map<String, RemovalSetting> removalSettings = ImmutableMap.of(
             "one", new RemovalSetting(
                     "Pokemon", RemovalType.WHITELIST, Lists.newArrayList("pixelmon:pixelmon"),
-                    Lists.newArrayList("!shiny", "boss:false"), true,
+                    Lists.newArrayList("shiny:false", "boss:notboss"), true,
```

Comment:
This should be left as !shiny

Reply:
What about the boss spec that doesn't matter?